src/index.js

```diff
@@ -0,0 +1,164 @@
+'use strict';
+
+//eevent happens (clicking on the get realtime temp)
+//take in city from city-input
+//request location api to return lat and lon, using city name
+//using the returned lat and lon, request weather api to return temperature
+//convert temp from kelvin to farhenheit
+//set that value as temp display
+const getTempFromCoordinates = () => {
+  const cityInput = document.getElementById('city-display').textContent;
+  console.log(cityInput);
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: cityInput,
+      },
+    })
+    .then((response) => {
+      console.log('success', response);
+      const return_obj = {
+        lat: response.data[0].lat,
+        lon: response.data[0].lon,
+      };
+      axios
+        .get('http://127.0.0.1:5000/weather', {
+          params: {
+            lat: return_obj['lat'],
+            lon: return_obj['lon'],
+          },
+        })
+        .then((weatherResponse) => {
+          console.log(weatherResponse);
+          const cityTemp = weatherResponse.data['main']['temp'];
+          let tempDisplay = document.getElementById('temp');
+          console.log(cityTemp);
+          const tempFahrenheit =
+            (parseInt(cityTemp) - parseInt(273.15)) * 1.8 + 32;
```

Comment:
I like this logic! However, we don't need to do `parseInt` on a literal we define ourself. We could either just have it as `273.15` or `273`, whichever we wanted. Also, consider moving this logic to a helper function.